.github/workflows/tests.yml

```diff
@@ -0,0 +1,30 @@
+name: rubocop-govuk
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.6
```

Comment:
Is there anyway we can get this version number from the `.ruby_version` in the app? Just to reduce number of places that need to change for a Ruby upgrade.

Though going off on a different tangent, one thing to bear in mind on a side note is that our Jenkins CI process tests our gems against past and newer ruby versions too. It's somewhat poorly maintained (I think it probably tests 2.4 and 2.5 Ruby rather than 2.7 for instance) but it is a nice practice for Gems. Is that relatively easy? Similar approach done with Travis: https://github.com/kevindew/openapi3_parser/blob/08cda87562cb676c8cf602f15359e63eb051e993/.travis.yml#L7-L12

Reply:
Yep, Actions calls them "matrix builds".

Reply:
We can do [both](https://github.com/ruby/setup-ruby#supported-version-syntax) those [things](https://github.com/ruby/setup-ruby#matrix).

Reply:
Feel free to push commits to try it out!

Reply:
Note that this is using the `ruby/setup-ruby` action rather than the GitHub `actions/setup-ruby` one as the latter can't pull `.ruby-version` files. But the `ruby/setup-ruby` action is in use [for the coronavirus forms](https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/pull/239).